installer.sh

```diff
@@ -104,7 +104,43 @@ case "$ID" in
                 #TPM2_TOOLS_PKGS="tpm2-tss tpm2-tools" TODO: still on 3.1.1 tpm2_tools
                 NEED_BUILD_TOOLS=1
                 NEED_PYTHON_DIR=1
-                POWERTOOLS="--enablerepo=PowerTools install autoconf-archive"
+		POWERTOOLS="--enablerepo=PowerTools install autoconf-archive"
```

Comment:
I have a couple of questions regarding this change. Firstly, if the only difference between CentOS and RHEL is this `POWERTOOLS` variable used in CentOS 8 / RHEL-8, wouldn't it be simpler to keep the original control structure and minimize the change something like:
```sh
case "$ID" in
  rhel | centos)
    case "${VERSION_ID}" in
      8*)
        # ...
        if test "$ID" = centos; then
          POWERTOOLS="..."
        else
          POWERTOOLS="..."
        fi
      ;;
    esac
  ;;
esac
```
Secondly, if this is all about enabling EPEL8, would it make sense to follow the officially suggested [instructions](https://docs.fedoraproject.org/en-US/epel/#_el8)?

Reply:
I agree with @ueno, I am trying to find differences but is the same code. Even `CENTOS7_TSS_FLAGS=` is in the `rhel` section.

Also seems that a tab is breaking the line.

Reply:
Thank you for your notes! This is about enabling the PowerTools repository (or for RHEL, codeready-builder). The current installler.sh fails and exits for RHEL 8 because this repository for RHEL is named differently ([source](https://access.redhat.com/discussions/5417621)). I have made changes to retain the original control structure and minimize the changes.